### DISCUSSION TITLE
question about  shadowtest.cpp

### DISCUSSION THREAD

**Discussion by Yan-Xiaodi** [google/filament#4205] [Q&A]
when I run with shadowtest project, I didn't see the monkey like this 
![image](https://user-images.githubusercontent.com/60952820/123573817-39219080-d801-11eb-9ebc-70110e3877a8.png)

but I got  this interface,I want to konw what‘s wrong with it.I haven't change any code yet. Thanks a lot
![image](https://user-images.githubusercontent.com/60952820/123573846-5191ab00-d801-11eb-9ba3-a576f73c826a.png)

My environment is: Visual Studio 2019

**Comment by romainguy** [comment]
Which version of Filament are you using?

@prideout @bejado Didn't we fix or see this exact bug recently?

**Reply by bejado** [reply]
I definitely remember seeing a similar bug, though I can't seem to find it.

@Yan-Xiaodi I just tried and I'm unable to reproduce the issue either on main or the commit you referenced. It'd also be helpful to know more about your system specs, in particular your GPU vendor and model.

**Reply by Yan-Xiaodi** [reply]
yes，your words remind me, the GPU in this computer is `Intel(R) HD Graphics 630 ` .  
it result:
![image](https://user-images.githubusercontent.com/60952820/123599480-38502500-d828-11eb-9584-564237a6ed67.png)


so I tried in annoher PC,which GPU is GTX1070,  it works
results that:
![image](https://user-images.githubusercontent.com/60952820/123599672-73525880-d828-11eb-9798-f27bd55ad829.png)

But I try the `material_sandbox`  in `Intel(R) HD Graphics 630 ` PC ,it can display successfully.

**Reply by jeanlemotan** [reply]
> Which version of Filament are you using?
> 
> 
> 
> @prideout @bejado Didn't we fix or see this exact bug recently?

There is another discussion with the same issue: 

https://github.com/google/filament/discussions/4021

**Comment by Yan-Xiaodi** [comment]
Thanks for your reply，my  version is this  main branch，relase in 06-08
![image](https://user-images.githubusercontent.com/60952820/123582537-8fe29680-d810-11eb-8f79-2f5be433aa12.png)
